### ISSUE TITLE
[bug]: Zoom is disabled on documentation

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5232]
### 🐞 Describe the bug

Because of this meta tag: `<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1">` in the head of the storybook documentation, the zoom is disabled on the docs. `maximum-scale` should be at a minimum of 5.

According to https://stackoverflow.com/questions/2989263/disable-auto-zoom-in-input-text-tag-safari-on-iphone max scale should not be used.

### Tasks
- [ ] Remove max scale

**Comment by alionazherdetska** [comment]
I have opened the issue on Storybook project: https://github.com/storybookjs/storybook/issues/31272

**Comment by alionazherdetska** [comment]
I have opened the issue on Storybook project: https://github.com/storybookjs/storybook/issues/31272 to address this issue.
